cpu/src/lib.rs

```diff
@@ -466,6 +482,58 @@ where
     }
 }
 
+fn index_of_word(addr: u32) -> usize {
+    return ((addr & !3) - 1) as usize;
+}
+
+impl<M, F> Instruction<M, F> for StoreU8Instruction
+where
+    M: MachineWithCpuChip<F>,
+    F: Field,
+{
+    const OPCODE: u32 = STOREU8;
+
+    fn execute(state: &mut M, ops: Operands<i32>) {
+        let opcode = <Self as Instruction<M, F>>::OPCODE;
+        let clk = state.cpu().clock;
+        let read_addr = (state.cpu().fp as i32 + ops.c()) as u32;
+        let write_addr_loc = (state.cpu().fp as i32 + ops.b()) as u32;
+        let pc = state.cpu().pc;
+        let write_addr = state
+            .mem_mut()
+            .read(clk, write_addr_loc.into(), true, pc, opcode, 0, "");
+
```

Comment:
`cell` should be read from `index_of_read`.

Reply:
I'm reading a whole word and then take the only relevant byte because each operations e.g., [Read](https://github.com/valida-xyz/valida/blob/09614eaf963b5d1ce5fb73c855dee650cd2d0d21/memory/src/lib.rs#L227) needs to fill in the [`value`](https://github.com/valida-xyz/valida/blob/09614eaf963b5d1ce5fb73c855dee650cd2d0d21/memory/src/columns.rs#L13) column which is a `word`. I tried to create a `read_byte` operation first but I found out that I will need to fill the byte back into a `word` for the `value` column so I thought I should just read and write the whole cell. What do you think?